Docs/Book/C++Examples/FindMyRDKit.cmake

```diff
@@ -0,0 +1,37 @@
+# Attempts to find RDKit libraries using the current value of $RDBASE
```

Comment:
The RDKit build system already creates and installs cmake configuration files (https://github.com/rdkit/rdkit/blob/master/CMakeLists.txt#L110) that are intended to make custom FindRDKit.cmake modules unnecessary.

I'm think these configuration files haven't been very much used, and there's probably space for improvements (one thing I recently thought could be nice to add, is exporting the build options, so that projects could automatically reconfigure based on the availability of optional RDKit modules), but I would suggest using this mechanism over distributing custom cmake modules/files.

Unfortunately, the only example I can easily provide is from chemicalite (https://github.com/rvianello/chemicalite/blob/master/CMakeLists.txt), but there's a tutorial on the cmake wiki that illustrates how to create/use these configuration files https://cmake.org/Wiki/CMake/Tutorials/How_to_create_a_ProjectConfig.cmake_file